main.go

```diff
@@ -11,15 +11,14 @@ import (
 	"path"
 	"time"
 
-	"github.com/go-logr/logr"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
Is this really needed? If it's for future proofing, should we import `"k8s.io/client-go/plugin/pkg/client/auth"` to support more comprehensive cloud providers authentication?

Reply:
It's not new, it's a relocation from lower in the file. It just got re-ordered. I agree we should create an issue to investigate whether we really need to load this package, but I don't think that should block _this_ PR.